src/result_generation/height.py

```diff
@@ -63,6 +65,7 @@ def __init__(
             workflows,
             artifact_workflow_path,
             scan_workflow_path,
+            scan_MCNN_workflow_path,
```

Comment:
I suggest to use lowercase in variable names. 
I suggest to use the existing experiment's name, so we don't have to remember another name for the same thing. 

```suggestion
            scan_depthmapmultiartifactlatefusion_workflow_path,
```